mtgjson5/providers/cardmarket/monolith.py

```diff
@@ -36,6 +31,7 @@ class CardMarketProvider(AbstractProvider):
 
     connection: Mkm
     set_map: Dict[str, Dict[str, Any]]
+    PRICE_GUIDE_URL: str = "https://downloads.s3.cardmarket.com/productCatalog/priceGuide/price_guide_1.json"
```

Comment:
Note to self: Probably worth moving to a config in case the URL changes over time, easier to manage